### DISCUSSION TITLE
EtherScan API Is invalid

### DISCUSSION THREAD

**Discussion by akashnikam25** [Cyfrin/foundry-full-course-cu#1698] [Q&A]
```
Encountered an error verifying this contract:
Response: `NOTOK`
Details: `Invalid API Key`
make: *** [deploy] Error 1

```
getting this error 
I have created etherscan API and provided but getting the above error.

**Comment by EngrPips** [comment]
Honestly, I would advise you to check that you have copied the correct `etherscan API key`
